src/main/daml/Daml/Finance/Interface/Types/Date/Calendar.daml

```diff
@@ -22,19 +22,19 @@ data HolidayCalendarData = HolidayCalendarData
 
 -- | An enum type to specify how a non-business day is adjusted.
 data BusinessDayConventionEnum
-  = FOLLOWING
+  = Following
   -- ^ Adjust a non-business day to the next business day.
-  | MODFOLLOWING
+  | Modfollowing
```

Comment:
Maybe ModFollowing?

Reply:
changed to `ModifiedFollowing` as discussed